core/ide/che-core-ide-jseditor/src/main/java/org/eclipse/che/ide/jseditor/client/debug/BreakpointManagerImpl.java

```diff
@@ -141,33 +136,15 @@ public void changeBreakpointState(final int lineNumber) {
      * Removes breakpoint mark.
      */
     private void deleteBreakpoint(final VirtualFile activeFile,
-                                  final Debugger debugger,
+                                  @Nullable final Debugger debugger,
```

Comment:
I'd rather write a good javadoc and describe all possible situations then mark each argument with such annotations. User may pass `null` anyway if you mark argument as `@NotNull`.